torch/csrc/utils/python_arg_parser.h

```diff
@@ -389,9 +390,51 @@ inline std::vector<int64_t> PythonArgs::intlist(int i) {
   return intlistWithDefault(i, signature.params[i].default_intlist);
 }
 
+static bool is_symint_node(py::handle obj) {
+      try {
+        auto sn = obj.cast<c10::SymbolicIntNode*>();
+        return true;
+      }
+      catch (...) {
+        return false;
```

Comment:
nuh uh, you're not committing this. Figure out a way to do it without exceptions